bin/tpaexec

```diff
@@ -1091,10 +1101,33 @@ esac
 # If we can find a venv located relative to TPA_DIR, we activate it as a
 # convenience.
 
+if [ -z ${PYTHON+x} ]; then
+    PYTHON=/usr/libexec/edb-python39/bin/python3
+    if [ ! -f $PYTHON ]; then
+        PYTHON=python3
+    else
+        echo "Running edb-python $PYTHON"
+    fi
+fi
+
 if [[ -f $TPA_VENV/bin/activate ]]; then
     _activate_venv "$TPA_VENV"
 fi
 
+case "$command" in
+    *help|info|version)
+        # Let through some basic commands, info in particular, without
+        # checking the Python version first.
+        ;;
+
+    *)
+        if [[ $(_python39_or_later) == "False" ]]; then
+            error "TPA requires Python 3.9+, but found only" \
+                "$(_python_version) ($(command -v $PYTHON))"
```

Comment:
Will this break if it couldn't find a python at all?

Reply:
Well. tpa relies on python so it'll be broken if no python is installed. In that case, we'll see an error on the output:

```
$ PYTHON=/bin/non-existing ./bin/tpaexec setup
./bin/tpaexec: line 87: /bin/non-existing: No such file or directory
Creating venv /home/mario/tpa/tpa-venv
./bin/tpaexec: line 119: /bin/non-existing: No such file or directory
ERROR: couldn't initialise /home/mario/tpa/tpa-venv using '/bin/non-existing -m venv'
```

Reply:
I think we could use a better error message there.

Reply:
I don't think that's what Abhijit was referring to. What happens if there is no python installed? What would the $PYTHON variable hold?

Reply:
tpaexec will check for edb-python otherwise $PYTHON will be python3 always.